### PR TITLE
Optimice installation 2.0

### PR DESCRIPTION
**Pull request by ipa-nhg**
solves merge conflict #69 and add some updates

### CODE REVIEW

File: InstallCob4.sh [hunk 1]
```diff
@@ -255,13 +254,18 @@ function Cob4Setup
...
+
   echo -e "\n${green}INFO:  Define users rights${NC}\n"
   sleep 5
-  sudo echo "%users ALL=NOPASSWD:/usr/sbin/cob-start" | sudo tee -a /etc/sudoers
-  sudo echo "%users ALL=NOPASSWD:/usr/sbin/cob-stop" | sudo tee -a /etc/sudoers
-  sudo echo "%users ALL=NOPASSWD:/usr/sbin/cob-stop-core" | sudo tee -a /etc/sudoers
- 
+
+
+grep -q -F '%users ALL=NOPASSWD:/usr/sbin/cob-start' /etc/sudoers  || sudo sh -c '(echo "%users ALL=NOPASSWD:/usr/sbin/cob-start") > /etc/sudoers'
```
Comment (ipa-fmw) [inline InstallCob4.sh]: identation

File: InstallCob4.sh [hunk 2]
```diff
@@ -255,13 +254,18 @@ function Cob4Setup
...
+
+
+grep -q -F '%users ALL=NOPASSWD:/usr/sbin/cob-stop' /etc/sudoers  || sudo sh -c '(echo "%users ALL=NOPASSWD:/usr/sbin/cob-stop") > /etc/sudoers'
```
Comment (ipa-fmw) [inline InstallCob4.sh]: identation

File: InstallCob4.sh [hunk 3]
```diff
@@ -255,13 +254,18 @@ function Cob4Setup
...
+
+
+ grep -q -F '%users ALL=NOPASSWD:/usr/sbin/cob-stop-core' /etc/sudoers  || sudo sh -c '(echo "%users ALL=NOPASSWD:/usr/sbin/cob-stop-core") > /etc/sudoers'
```
Comment (ipa-fmw) [inline InstallCob4.sh]: identation

File: InstallCob4.sh [hunk 4]
```diff
@@ -272,7 +276,47 @@ function Cob4Setup
...
+function Mimic
+{
+  if [ "$HOSTNAME" != "$ROBOT-h1" ]; then 
+	  echo "FATAL: CAN ONLY BE EXECUTED ON Slave h1 PC"
+	  exit
+  fi
+ echo -e "\n${green}INFO:  Mimic ${NC}\n"
+  sleep 5
+
+sudo ssh $server "cob-adduser mimic"
```
Comment (ipa-fmw) [inline InstallCob4.sh]: identation (also for following lines)

File: InstallCob4.sh (hunk 1 above)
Comment (ipa-nhg) [inline InstallCob4.sh]: these lines should be removed, my error during merging

**Pull request merged**